oozie/src/main/java/org/apache/falcon/oozie/feed/FeedReplicationWorkflowBuilder.java

```diff
@@ -145,6 +161,24 @@ private String getDefaultMapBandwidth() {
         return RuntimeProperties.get().getProperty("falcon.replication.workflow.mapbandwidth", "100");
     }
 
+    private String getDefaultAmMemory() {
```

Comment:
Still its better to move it to OrchestrationWorkflowBuilder to avoid repetition of code.We can override the method in  ProcessWorfklowBuilder and throw an exception method not supported if donot want to use them there.